### PR TITLE
Batch resource creation file path to function definition and CLI output

### LINKED ISSUE #295: Accept path to function from batch create/update
Creating functions via manifest file (`examples/seed.yaml`), the function needs to be entirely defined inline.  This is less than ideal.  We should also accept a file path to the function `@pathToFile.py`

- [ ] Bug
- [ ] Feature
- [x] Enhancement

## Possible Implementation

The client should follow the path which is identified by the following panther `@pathToFile.py` (similar to curl), and unpack the file.  Then the client sends the contents as part of the payload to the server.

## Complexity

- [x] Low - Simple enhancement or bug fix, no architectural changes or refactoring
- [ ] Medium - Change requires some thought, but is relatively isolated
- [ ] High - Significant architectural change or large refactor

## Impact

- [x] Low - Annoyance, but does not impact business or functionality
- [ ] Medium - Issue can be worked around, but is causing pain
- [ ] High - Blocker

## Your Environment

* Dispatch CLI version (or git commit):
* Dispatch Chart version (or image[s] tag):
* Operating System and version:
* Kubernetes version (and distribution):
* Etc (any other useful environmental information):

### PR DESCRIPTION
**Pull request by rjew**
Fixes #295 and #288 

* Resource type and name are outputted when created via manifest file
* Function definition can be specified in manifest as a path to file with notation `@pathToFile.py`

### CODE REVIEW

**Comment by kars7e** [comment]
Super cool!

File: examples/seed.yaml [hunk 1]
```diff
@@ -82,14 +56,7 @@ tags:
 ---
 kind: Function
 name: http-py
-code: |
```
Comment (tenczar) [inline examples/seed.yaml]: Could we leave one of these as the function 'literal' to show that we support both?

File: pkg/dispatchcli/cmd/create.go [hunk 2]
```diff
@@ -105,17 +109,28 @@ func importFile(out io.Writer, errOut io.Writer, cmd *cobra.Command, args []stri
...
 			o.Images = append(o.Images, m)
+			fmt.Fprintf(out, "Created %s: %s\n", docKind, *m.Name)
 		case utils.FunctionKind:
 			m := &functionModels.Function{}
 			err = yaml.Unmarshal(doc, &m)
 			if err != nil {
 				return errors.Wrapf(err, "Error decoding function document %s", string(doc))
 			}
+			if strings.HasPrefix(*m.Code, "@") {
+				functionPath := (*m.Code)[1:]
```
Comment (tenczar) [inline pkg/dispatchcli/cmd/create.go]: Shouldn't this be modified to work with the work-dir flag?

Comment (berndtj) [reply pkg/dispatchcli/cmd/create.go]: Could you create a new issue?  Also, I'd like path supported for dependency files (e.g. requirements.txt/package.json)

**Pull request merged**